app/src/main/res/values/strings.xml

```diff
@@ -422,4 +423,6 @@
   </string>
   <string name="smallest_text_size_content_description">Smallest text size</string>
   <string name="largest_text_size_content_description">Largest text size</string>
+  <string name="unlock_chapter">"Complete <![CDATA[<b>Chapter %s: %s</b>]]> to unlock this chapter."</string>
+  <string name="chapter_land_name"><![CDATA[<b>Chapter %s: %s</b>]]></string>
```

Comment:
This is incorrect. Making the text bold is not same as `Roboto-Medium`.

Reply:
@rt4914 I agree, but I was not sure whether I can use Roboto-Medium or not, as it's an external font

Reply:
This solution will work for now.